spec/controllers/conference_registration_controller_spec.rb

```diff
@@ -23,12 +23,33 @@
     end
   end
 
+  shared_examples 'create registration' do |user, ichain|
```

Comment:
This set of tests is more about 'cannot access #new action', as you are not testing the creation of a registration, but whether or not a user can access the new action for a registration. So I would change the description of the shared examples to reflect what they are doing.